### DISCUSSION TITLE
Question:Why Isn't The Download For Week 7 Available?

### DISCUSSION THREAD

**Discussion by idragons962** [FunkinCrew/Funkin#1564] [General]
Why Isn't The Download Available

**Comment by ghost** [comment]
go to itch.io is it that hard lmao

**Comment by idragons962** [comment]
No The Week 7 Update

**Comment by ghost** [comment]
@idragons962
ah

**Comment by idragons962** [comment]
I Guess It's Not Available Case Closed.

**Comment by idragons962** [comment]
No The Week 7 Download

**Comment by idragons962** [comment]
https://github.com/idragons962/Fnf-Corruption-mod

**Reply by GuglioREAL** [reply]
mans just promoted his mod here

**Comment by ghost** [comment]
Week 7 is not downloadable because
1. they cant get cutscenes working
2. ng needs to be 1st or 2nd

**Reply by AbnormalPoof** [reply]
1. false, they're most likely tackling a different approach since there are spritesheets in the files
2. already happened

**Comment by Windows1Door2Daniel** [comment]
for now is NG exclusive

**Comment by xenten9** [comment]
It is only available on New Grounds as an exclusive so if you want to play it you will have to go there.

**Comment by ACrazyTown** [comment]
close

**Comment by Windows1Door2Daniel** [comment]
> close

A side of comment button says close issue i think
